packages/RichTextEditing-Core.package/DocumentReadStream.class/instance/on..st

```diff
@@ -2,5 +2,5 @@ initialize
 on: aDocument
 
 	collection := aDocument text.
-	readLimit := collection size.
+	self readLimit: collection size.
```

Comment:
We did not use a setter because you should not be able to change the readlimit after opening a stream

Reply:
Ok fine, but then remove the method